FiveCalls/FiveCalls/IssueDetailViewController.swift

```diff
@@ -24,13 +24,34 @@ class IssueDetailViewController : UIViewController, IssueShareable {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-                
+        print("vdl delegate\(self.tableView.delegate)")
+
         navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .action, target: self, action: #selector(IssueDetailViewController.shareButtonPressed(_ :)))
         
         tableView.estimatedRowHeight = 100
         tableView.rowHeight = UITableViewAutomaticDimension
         NotificationCenter.default.addObserver(self, selector: #selector(madeCall), name: .callMade, object: nil)
     }
+
+    func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
+        guard section == IssueSections.header.rawValue else {
+            return nil
+        }
+        let view = UIView(frame: CGRect(x: 0, y: 0, width: tableView.frame.width, height: 40.0))
+        let label = UILabel(frame: CGRect(x: 16, y: 0, width: tableView.frame.width - 32.0, height: 40))
+        let button = UIButton(frame: CGRect(x: 0, y: 0, width: tableView.frame.width, height: 40))
+        view.addSubview(label)
+        view.addSubview(button)
+
+        view.backgroundColor = UIColor(white: 245.0/255, alpha: 1.0)
```

Comment:
Can you define this color where the rest of our colors are defined? Then use `.fvc_grayColor()` or something here.